src/a5dd0_len_114e0.c

```diff
@@ -5690,87 +5689,70 @@ Gfx* mdl_get_copied_gfx(s32 copyIndex) {
     return gfxCopy;
 }
 
-#ifdef WIP
-void mdl_project_tex_coords(s32 modelID, Gfx* destGfx, f32 (*destMtx)[4], void* destVertices) {
-    s32 numVertices;
+void mdl_project_tex_coords(s32 modelID, Gfx* arg1, Matrix4f arg2, Vtx* arg3) {
+    s32 sp18;
     Vtx* baseVtx;
-    s32 gfxCount;
+    s32 sp20;
+    f32 v1tc1;
+    f32 v2tc1;
     f32 sp2C;
+    f32 v0tc1;
     f32 sp40;
-    Vtx* temp_a0;
-    f32 temp_f10;
-    f32 temp_f12;
-    f32 temp_f12_2;
-    f32 temp_f14;
-    f32 temp_f14_2;
-    f32 temp_f2;
-    f32 temp_f2_2;
-    f32 temp_f4_2;
-    f32 temp_f4_3;
-    f32 temp_f8;
-    f32 temp_f8_3;
-    f32 var_f0;
-    f32 var_f10;
-    f32 var_f24;
-    f32 var_f26;
-    f32 var_f2;
-    f32 var_f30;
-    f32 var_f6;
-    f32 var_f6_2;
-
+    f32 v1tc0;
+    f32 v1ob2;
+    f32 ob2;
+    f32 ob1;
     f32 v0ob0;
     f32 v0ob2;
     f32 v0tc0;
-    f32 v0tc1;
-
-    f32 v1ob0;
-    f32 v1ob2;
     f32 v2ob0;
-    f32 v1tc0;
-    f32 v1tc1;
-    f32 v2ob2;
     f32 v2tc0;
-    f32 v2tc1;
-
+    f32 v1ob0;
+    f32 v2ob2;
     f32 ob0;
-    f32 ob1;
-    f32 ob2;
-
-    f32 tc0;
+    f32 var_f10;
+    f32 var_f24;
+    f32 var_f26;
     f32 tc1;
-
-    s32 cn0;
-    s32 cn1;
-    s32 cn2;
-    s32 cmd;
+    f32 var_f30;
+    f32 tc0;
+    f32 var_f6_2;
     s32 i;
+    u32 cnB;
+    u32 cnG;
+    u32 cnR;
+    f32 var_f20;
 
     s32 listIndex;
     Model* model;
     Gfx* dlist;
+    s32 cmd;
+    Vtx* tempVert;
+
+    s8 zero; // TODO needed to match
 
     listIndex = get_model_list_index_from_tree_index(modelID & 0xFFFF);
     model = get_model_from_list_index(listIndex);
     dlist = model->modelNode->displayData->displayList;
+    zero = 0;
```

Comment:
We're certain `zero` needs to be given a value after these func calls and not at declaration?